src/pilot/deploy-overcloud.py

```diff
@@ -663,18 +310,12 @@ def main():
                             action='store_true',
                             help="Indicates if the deploy-overcloud script "
                                  "should be run in debug mode")
-
-        LoggingHelper.add_argument(parser)
-
         args = parser.parse_args()
 
-        LoggingHelper.configure_logging(args.logging_level)
```

Comment:
Removing the above line breaks the ability to pass in a command line argument to set the logging level.

Reply:
Required added